### DISCUSSION TITLE
client.channelList에서 channelId와 방 이름 구하기

### DISCUSSION THREAD

**Discussion by bass9030** [storycraft/node-kakao#441] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/59216834/123513150-71708400-d6c6-11eb-92a8-1251130da30b.png)
![image](https://user-images.githubusercontent.com/59216834/123513161-80573680-d6c6-11eb-8249-3505eb19e878.png)
카카오톡에 있는 방들의 이름과 channelId를 구하고 싶습니다.
`client.channelList.size`를 하면 방 갯수가 잘 표시되는데, `JSON.stringify(client.channelList.all())`를 하면 사진과 같이 허전한 JSON이 반환됩니다.
혹시 제가 접근한 방식으로 접근하는것이 아닌건가요?
가능하면 간단한 예제와 같이 답변 부탁드립니다..

**Comment by wiresharkkkk** [comment]
방이름 구하기 저도 찾고있었는데 혹시 해결하셨나요?

**Comment by ghost** [comment, marked as answer]
```ts
Array.from(client.channelList.all()).map(e => ({ name: e.getDisplayName(), id: e.channelId }));
```

**Reply by bass9030** [reply]
감사합니다!
